gpii/configs/production.with.logging.json

```diff
@@ -22,7 +22,7 @@
             }
         }
     },
-    "includes": [
+    "mergeConfigs": [
         "../node_modules/deviceReporter/configs/production.json",
```

Comment:
this URL is wrong, should be `../node_modules/deviceReporter/configs/gpii.deviceReporter.config.production.json`... But tbh, I think perhaps this config should just be removed since there already is a production config that is even more production'y than this one